commit/merkleroot/outcome.go

```diff
@@ -211,29 +212,23 @@ func getConsensusObservation(
 ) (ConsensusObservation, error) {
 	aggObs := aggregateObservations(aos)
 
-	fMin := make(map[cciptypes.ChainSelector]int)
-	for chain := range aggObs.FChain {
-		fMin[chain] = F
-	}
 	// consensus on the fChain map uses the role DON F value
 	// because all nodes can observe the home chain.
-	fChains := plugincommon.GetConsensusMap(lggr, "fChain", aggObs.FChain, fMin)
+	donThresh := consensus.MakeConstantThreshold(consensus.TwoFPlus1(F))
+	fChains := consensus.GetConsensusMap(lggr, "fChain", aggObs.FChain, donThresh)
 
 	_, exists := fChains[destChain]
 	if !exists {
 		return ConsensusObservation{},
 			fmt.Errorf("no consensus value for fDestChain, destChain: %d", destChain)
 	}
 
-	twoFPlus1 := make(map[cciptypes.ChainSelector]int)
-	for chain, f := range fChains {
-		twoFPlus1[chain] = 2*f + 1
-	}
-
+	// Get consensus using strict 2f+1 threshold.
+	twoFPlus1 := consensus.MakeMultiThreshold(fChains, consensus.TwoFPlus1)
 	consensusObs := ConsensusObservation{
-		MerkleRoots:        plugincommon.GetConsensusMap(lggr, "Merkle Root", aggObs.MerkleRoots, fChains),
-		OnRampMaxSeqNums:   plugincommon.GetConsensusMap(lggr, "OnRamp Max Seq Nums", aggObs.OnRampMaxSeqNums, twoFPlus1),
-		OffRampNextSeqNums: plugincommon.GetConsensusMap(lggr, "OffRamp Next Seq Nums", aggObs.OffRampNextSeqNums, fChains),
+		MerkleRoots:        consensus.GetConsensusMap(lggr, "Merkle Root", aggObs.MerkleRoots, twoFPlus1),
+		OnRampMaxSeqNums:   consensus.GetConsensusMap(lggr, "OnRamp Max Seq Nums", aggObs.OnRampMaxSeqNums, twoFPlus1),
+		OffRampNextSeqNums: consensus.GetConsensusMap(lggr, "OffRamp Next Seq Nums", aggObs.OffRampNextSeqNums, twoFPlus1),
```

Comment:
MerkleRoots and OffRampNextSeqNums were using a threshold of `fChain` rather than `2*fChain+1`.

Reply:
oops :D